README.md

```diff
@@ -4,4 +4,4 @@
 
 ## License
 
-MIT © [AgFlow](https://www.agflow.com)
+MIT © [AgFlow](https://www.agflow.com) <img border="0" alt="AgFlow" src="https://raw.githubusercontent.com/agflow/logos/master/agflow-logo.png" width="140" height="100">
```

Comment:
doesn't look nice

maybe put below line and give more width or less height.

Reply:
OK, I'm gonna fix all of them. Are we gonna add logo in "danger-plugin-no-eslint-disable " ?

Reply:
yes, why not ?

However, first close open ones, we can do it afterwards.